build.sbt

```diff
@@ -71,6 +71,10 @@ lazy val codegen = Project(id = "codegen", base = file("codegen"))
     (assembly / mainClass) := Some("org.apache.pekko.grpc.gen.Main"),
     (assembly / assemblyOption) := (assembly / assemblyOption).value.withPrependShellScript(
       Some(sbtassembly.AssemblyPlugin.defaultUniversalScript(shebang = true))),
+    (assembly / assemblyMergeStrategy) := {
+      case PathList("META-INF", _*) => MergeStrategy.discard
+      case _                        => MergeStrategy.first
```

Comment:
Wouldn't it be safer to use `MergeStrategy.deduplicate`? (https://github.com/sbt/sbt-assembly?tab=readme-ov-file#merge-strategy, also https://arxiv.org/abs/2407.18760 though I'm more concerned about correctness/reproducibility than with security in this case)

Reply:
changed